notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb

```diff
@@ -122,6 +122,8 @@
     "from azureml.core.image import ContainerImage\n",
```

Comment:
<p>cpu_cores is a little confusing to me (being a fraction), how should I think about what that number means? where did 0.3 come from?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/735/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
it's the number of cores you want to allocate per replica, and since you can have multiple replicas / agent, it can be fractional. See [here](https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#cpu-units) for more info.

Reply:
and agreed - it is non-intuitive

Reply:
got it, I was thinking of replicas as separate vms, not containers on the same vm.

Reply:
<p>updated with more explanation and link to kube docs.</p>